README.md

```diff
@@ -1,2 +1,118 @@
-# lorawan-gateway
-viam module for a sx1302 lorawan gateway.
+# LoRaWAN Module
+
+Viam module for receiving data from LoRaWAN sensors.\
+`sx1302-gateway`: sensor model for a sx1302 lorawan concentrator hat connected to a raspberry pi.\
+`node`: sensor model for the end nodes sending data to the gateway.
```

Comment:
Write a little bit about LoRaWan, what it is and what this module ultimately does. Also, please follow the template in the viam docs for the first parts of your readme.